### DISCUSSION TITLE
Getting ORA-12541: Cannot connect. No listener at host localhost port 1521 with colima (macos M3)

### DISCUSSION THREAD

**Discussion by morenohernan** [abiosoft/colima#953] [Q&A]
Hi, 
I'm starting with colima as an alternative to use docker with oracle 12c in mac M1/M3
The setup was straightforward and everything was working great, meaning that I've configured the oracle 12c db populated several tables connected to my app, etc etc. After stopping the container and colima and rebooting the laptop (and starting again everything including colima and the container) I'm getting ORA-12541: Cannot connect. No listener at host localhost port 1521 (from the host..  meaning the mac/ my app running locally)

Checking the following:
the status of the db looks healthy (docker ps)
<img width="1727" alt="image" src="https://github.com/abiosoft/colima/assets/700619/05a727fc-8258-4245-a432-876f270b61f0">

Making a lsnrctl status shows that all is good:
<img width="1727" alt="image" src="https://github.com/abiosoft/colima/assets/700619/39186205-57b3-4ca4-b586-ac77af6cc2f2">

<img width="755" alt="image" src="https://github.com/abiosoft/colima/assets/700619/9e0ed0d7-d519-4662-9bb1-9e2241ee24ad">

<img width="693" alt="image" src="https://github.com/abiosoft/colima/assets/700619/ba27ae42-8560-487b-bec3-b8595f8eb283">

The commands I've used were:
colima start 
docker start <container-name>

As other alternatives I've tried stopping and starting again all. Also with this option : colima start --network-address
But same result....

Inside the container and SQL I can make queries, etc but from outside (host machine) the port looks closed. Any hint?

**Comment by morenohernan** [comment]
By the way. Just as a test I've reinstalled another fresh db (different volume, etc) and the results are the same. It seems something going not related with the db

**Reply by bulalak** [reply]
Hello, I am encountering the same issue as described in this discussion. Has there been any progress on resolving this, or are there any recommended workarounds available? Any updates would be greatly appreciated

**Comment by abiosoft** [comment]
Can you try listening on another port, do you get same behaviour?

**Comment by catachimerel** [comment]
Has anybody managed to solve this issue? I am facing the exact same problem and can't figure out the cause.

**Reply by bulalak** [reply]
As a workaround, it works correctly after I downgraded Colima to version 0.5.6 by following these steps: https://github.com/abiosoft/colima/issues/877#issuecomment-1815402849.
